src/test/codewhisperer/explorer/codewhispererNode.test.ts

```diff
@@ -50,6 +51,20 @@ describe('codewhispererNode', function () {
             assert.strictEqual(treeItem.collapsibleState, vscode.TreeItemCollapsibleState.Collapsed)
         })
 
+        it('should create a node showing IAM connection', function () {
```

Comment:
this test and its siblings are almost identical, this would be the right time to de-duplicate them